pkg/scheduler/algorithm/priorities/requested_to_capacity_ratio_test.go

```diff
@@ -239,3 +239,381 @@ func TestRequestedToCapacityRatio(t *testing.T) {
 		}
 	}
 }
+func TestResourceBinPackingSingleExtended(t *testing.T) {
+	extendedResource := "intel.com/foo"
+	extendedResource1 := map[string]int64{
+		"intel.com/foo": 4,
+	}
+
+	extendedResource2 := map[string]int64{
+		"intel.com/foo": 8,
+	}
+
+	noResources := v1.PodSpec{
+		Containers: []v1.Container{},
+	}
+	extendedResourcePod1 := v1.PodSpec{
+		Containers: []v1.Container{
+			{
+				Resources: v1.ResourceRequirements{
+					Requests: v1.ResourceList{
+						v1.ResourceName(extendedResource): resource.MustParse("2"),
+					},
+				},
+			},
+		},
+	}
+	extendedResourcePod2 := v1.PodSpec{
+		Containers: []v1.Container{
+			{
+				Resources: v1.ResourceRequirements{
+					Requests: v1.ResourceList{
+						v1.ResourceName(extendedResource): resource.MustParse("4"),
+					},
+				},
+			},
+		},
+	}
+	machine2Pod := extendedResourcePod1
+	machine2Pod.NodeName = "machine2"
+	tests := []struct {
+		pod          *v1.Pod
+		pods         []*v1.Pod
+		nodes        []*v1.Node
+		expectedList schedulerapi.HostPriorityList
+		name         string
+	}{
+		{
+			/*
```

Comment:
block comments are typically used for package headers, please use // for comments within code.

Reply:
👍